test/jdk/sun/security/util/DerValue/Indefinite.java

```diff
@@ -23,9 +23,9 @@
 
 /*
  * @test
- * @bug 6731685 8249783
+ * @bug 6731685 8249783 8255393
```

Comment:
I don't think you need to extend the bug tag for this issue as it doesn't add/change the test.